docs/admin/records.md

```diff
@@ -0,0 +1,76 @@
+# Records
+
+## Hubspot Records
+
+Records are profiles of each transit agency or organization. Records contain general information about the agency, contact information for agency workers and the ability to track emails, calls, notes and tickets with that agency.
+
+When working with the CRM you may find that records need to be updated, merged or deleted to reflect the correct information. Some use cases may include transit agency mergers, duplicate records or other organizational updates. Some updates, deletions, or mergers may require research or corroborating with other Cal-ITP documents. All users are encouraged to highlight and update records so that they can reflect the most up to date information.
+
+### How are records created
+
+Records are created through two processes: data imports and when a user establishes communication with a contact at a new organization not yet recorded in the CRM. At this time, most new records will be emerging automatically from new communication over historical documents.
+
+### Creating a new company record
+
+New companies should only be established when they are not a part of any other agency that exists. Companies that are branded services, contracted out or operated by larger agencies should be associated with their larger city agencies. More information on associations can be found in the Updating Records section.
+
+## Updating Records
+
+Each record contains contact information for the agency and individuals at that agency, you may discover that information is not up to date, has missing information or incorrect information.
+
+## Use Cases for CRM Management
+
+Is a company record named after the official name of an organization, any “d/b/a” (doing business as) name it might have, the name of the service it provides, or any other branding it might be well known by?
```

Comment:
```suggestion
_Is a company record named after the official name of an organization, any “d/b/a” (doing business as) name it might have, the name of the service it provides, or any other branding it might be well known by?_
```